extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -35,17 +35,14 @@
 }
 
 .i-amphtml-story-interactive-slider-bubble {
```

Comment:
Adding `height: 100% !important;` will fix the vertical positioning.

Reply:
Without the transform on line 101 the alignment is perfect, but you lose the scaling

Reply:
`height: 100% !important;` is only for vertical alignment. 
Horizontal still needs to be figured out.

Good point about the scale. It might have something to do with the x transform of the scale.